config/environment.js

```diff
@@ -29,6 +37,14 @@ module.exports = function (environment) {
     // ENV.APP.LOG_TRANSITIONS = true;
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
+    ENV.acmidm = {
```

Comment:
Minor remark, when you implement the real login you should replace this by real data